cmd/soroban-rpc/internal/db/sqlmigrations/03_events.sql

```diff
@@ -0,0 +1,23 @@
+-- +migrate Up
+
+-- indexing table to find events in ledgers by contract_id
+CREATE TABLE events
+(
+    id                TEXT PRIMARY KEY,
+    contract_id       BLOB(32),
+    event_type        INTEGER NOT NULL,
+    event_data        BLOB    NOT NULL,
+    ledger_close_time INTEGER NOT NULL,
```

Comment:
wait why do we need this field denormalized?

Reply:
because we need `LedgerClosedAt` in `EventInfo` response. No other way to get it afaik as we dont rely on LCM anymore.

Reply:
Ah yes of course, good call